states/WV/governor.yaml

```diff
@@ -0,0 +1,46 @@
+contact_form:
+  steps:
+    - visit: "https://governor.wv.gov/Pages/SubmitaCommenttotheGovernor.aspx"
+    - fill_in:
+      - name: "ctl00$PlaceHolderMain$g_1aa949c5_afad_4f93_9c5b_1c3bf27389df$ff1_1$ctl00$ctl00$TextField"
+        selector: "#ctl00_PlaceHolderMain_g_1aa949c5_afad_4f93_9c5b_1c3bf27389df_ff1_1_ctl00_ctl00_TextField"
+        value: $NAME_FULL
+        required: true
+      - name: "ctl00$PlaceHolderMain$g_1aa949c5_afad_4f93_9c5b_1c3bf27389df$ff3_1$ctl00$ctl00$TextField"
+        selector: "#ctl00_PlaceHolderMain_g_1aa949c5_afad_4f93_9c5b_1c3bf27389df_ff3_1_ctl00_ctl00_TextField"
+        value: $ADDRESS_STREET
+        required: true
+      - name: "ctl00$PlaceHolderMain$g_1aa949c5_afad_4f93_9c5b_1c3bf27389df$ff4_1$ctl00$ctl00$TextField"
+        selector: "#ctl00_PlaceHolderMain_g_1aa949c5_afad_4f93_9c5b_1c3bf27389df_ff4_1_ctl00_ctl00_TextField"
+        value: $ADDRESS_STREET_2
+        required: false
+      - name: "ctl00$PlaceHolderMain$g_1aa949c5_afad_4f93_9c5b_1c3bf27389df$ff5_1$ctl00$ctl00$TextField"
+        selector: "#ctl00_PlaceHolderMain_g_1aa949c5_afad_4f93_9c5b_1c3bf27389df_ff5_1_ctl00_ctl00_TextField"
+        value: $ADDRESS_CITY
+        required: true
+      - name: "ctl00$PlaceHolderMain$g_1aa949c5_afad_4f93_9c5b_1c3bf27389df$ff6_1$ctl00$ctl00$TextField"
+        selector: "#ctl00_PlaceHolderMain_g_1aa949c5_afad_4f93_9c5b_1c3bf27389df_ff6_1_ctl00_ctl00_TextField"
+        value: $ADDRESS_STATE_FULL
```

Comment:
We usually use the state abbreviation whenever possible.

Reply:
Fixed.